feature-libs/organization/administration/components/shared/table/cell.component.ts

```diff
@@ -37,9 +47,15 @@ export class CellComponent {
    * If the cells is linkable, an anchor link is created to the detailed route
    * of the given `_type`.
    *
-   * Defaults to `true`.
+   * Defaults to `false`.
```

Comment:
I think we should clarify that it only defaults to false while the feature toggle is enabled. At least, until we can clean up breaking changes.

Reply:
I've edited the comment to make that clear.